system-setup.py

```diff
@@ -30,6 +30,9 @@ def redhat_compat(self):
         self.install("redhat-lsb-core")
         self.install("libatomic")
 
+        if self.dist == "centos" and self.ver == "8":
+            return
```

Comment:
If we bail out here, we won't have gcc installed.

Reply:
For centos-8, we don't need to. We have a modern gcc installed that compiles everything. Or - is the worry about specific gccs?

centos8 doesn't (yet) have the full backhistory of ports made.

Reply:
So `getgcc` is an abstraction of Ubuntu/Debian `apt install build-essential` and CentOS `yum groupinstall 'Development Tools'`, with a "modern" variation for those distributions that need it. So for CentOS 8, we can exclude `--modern` either from the `system-setup.py` side or from `getgcc`. I prefer the latter (as it will hold for all modules). We do need to invoke some sort of `getgcc`, otherwise no gcc will be installed.